README.md

```diff
@@ -32,3 +32,8 @@ python app.py
 Run the converter by selecting any of the files from test_files folder.
 Select any of the dropdown options.
 Hit submit and download button.
+
+## UnitTesting
+```
+python tests.py
```

Comment:
Let's use the more common CLI as specified here: https://docs.python.org/3/library/unittest.html#command-line-interface

Reply:
Sounds good working on it.